src/client/api/container.c

```diff
@@ -203,6 +171,11 @@ daos_cont_open2(daos_handle_t poh, const char *cont, unsigned int flags,
 	return dc_task_schedule(task, true);
 }
 
+int
+daos_cont_open2(daos_handle_t poh, const char *cont, unsigned int flags,
+		daos_handle_t *coh, daos_cont_info_t *info, daos_event_t *ev)
+		__attribute__ ((weak, alias("daos_cont_open")));
```

Comment:
(style)  Missing a blank line after declarations

Reply:
better fix it need be refreshed.

Reply:
https://github.com/daos-stack/daos/blob/lei_DAOS-9199/src/client/api/container.c#L178
There is a blank line there.

Reply:
This is just one of those where checkpatch misinterprets the code, I think.

Reply:
Not quite, it's the __attribute__ that comes after the declaration.  I believe you can silence this by putting the __attribute__ between the return type and the function name, but that can cause problems with some compilers.